src/components/AddRepoModal/RepoSearchResultList.tsx

```diff
@@ -2,8 +2,14 @@ import { RepoSearchResultCard } from '@/components/AddRepoModal/RepoSearchResult
 
 export function RepoSearchResultList({
   filteredResults,
+  trackedRepoIds,
+  setTrackedRepos,
+  setCurrentRepo,
 }: {
   filteredResults: GitHubRepo[];
+  trackedRepoIds: number[];
+  setTrackedRepos: React.Dispatch<React.SetStateAction<GitHubRepo[]>>;
+  setCurrentRepo: React.Dispatch<React.SetStateAction<GitHubRepo[]>>;
 }) {
   return (
     <div className="flex flex-col gap-4 mt-4">
```

Comment:
When running the branch locally, I added something like this to fix the search input at the top of the modal while the results scroll. The 300px for max-height is just a random value. 
```suggestion
    <div className="flex flex-col gap-4 mt-4 max-h-[300px] overflow-y-auto">
```